great_expectations_provider/__init__.py

```diff
@@ -0,0 +1,7 @@
+def get_provider_info():
+    return {
+        "package-name": "airflow-provider-great-expectations",
+        "name": "Great Expectations Provider",
+        "description": "An Apache Airflow provider for GE.",
```

Comment:
```suggestion
        "description": "An Apache Airflow provider for Great Expectations.",
```

Reply:
Style guide, we spell it out in docs (abbreviation is OK in inline code comments I guess)